src/test/regress/bin/normalize.sed

```diff
@@ -65,3 +69,19 @@ s/ERROR:  failed to execute task [0-9]+/ERROR:  failed to execute task X/g
 # normalize file names for partitioned files
 s/(task_[0-9]+\.)[0-9]+/\1xxxx/g
 s/(job_[0-9]+\/task_[0-9]+\/p_[0-9]+\.)[0-9]+/\1xxxx/g
+
+# Line info varies between versions
+/^LINE [0-9]+:.*$/d
+/^\s*\^\s*$/d
+
+# Ignore file paths
+s,"/(home|usr|lib)/(.+/)*([^/]*\.so)","\3",
+
+# pg12 changes
+s/Partitioned table "/Table "/g
+s/\) TABLESPACE pg_default$/\)/g
+s/invalid input syntax for type /invalid input syntax for /g
```

Comment:
There seems to be one more occurrence of `type` diffs on `worker_check_invalid_arguments.out`, so we could add that.

```
-ERROR:  invalid input syntax for integer: "\x0b50"
+ERROR:  invalid input syntax for type integer: "\x0b50"
```